src/Components/Footer/styles.js

```diff
@@ -0,0 +1,17 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  footerStyle: {
```

Comment:
Rename this to `footer`. Since you will use like `styles.footer`, there is no need to repeat the word style. Same for textStyle